templates/profile.html

```diff
@@ -23,7 +72,7 @@ <h2 class="mb-4 pb-4 border-b-2 inline-block w-3/4">{{auth.student_classes}}</h2
    <h1 class="bg-blue-200 px-8 py-2 mb-4 rounded-lg text-3xl cursor-pointer" onclick="hedyApp.load_profile ('{{user_data['username']}}', '{{user_data['email']}}', '{{user_data['birth_year']}}', '{{user_data['gender']}}', '{{user_data['country']}}')">{{auth.profile}}</h1>
```

Comment:
Note that the {{auth.profile}} is used here but it is not defined in the translations. As a result I see 'Unknown Exception' on the /my-profile page:
![Screenshot 2022-01-24 at 12 11 30](https://user-images.githubusercontent.com/3010723/150764028-368d2ce9-dcea-4dc8-80d8-ae417784b497.png)

Reply:
I think this is a merge issue as I solve this issue earlier. But will fix it again!